src/main/kotlin/blackjack/domain/Hand.kt

```diff
@@ -1,23 +1,22 @@
 package blackjack.domain
 
 class Hand(
-    val cards: MutableList<Card> = mutableListOf()
+    private val _cards: MutableList<Card> = mutableListOf()
 ) {
+    val cards: List<Card>
+        get() = _cards
+
     fun init(card1: Card, card2: Card) {
-        cards.add(card1)
-        cards.add(card2)
+        _cards.add(card1)
+        _cards.add(card2)
```

Comment:
```suggestion
        _cards.addAll(card1, card2)
```

addAll을 사용하는 건 어떨까요?

추가로, init에 card1, card2를 받는 것보다 cards :List<Card> 를 받고, init 함수에서 cards가 2장인가에 대한 밸리데이션을 할 수 있어보이는데, 어떻게 생각하시나요?

Reply:
> addAll을 사용하는 건 어떨까요?  

addAll을 사용하는 방향으로 수정해볼게요!

> 추가로, init에 card1, card2를 받는 것보다 cards :List 를 받고, init 함수에서 cards가 2장인가에 대한 밸리데이션을 할 수 있어보이는데, 어떻게 생각하시나요?  

cards :List을 인자로 받고, init 함수 내에서 유효성 검사를 진행하는게 좋을 것 같네요. 만약 블랙잭 규칙이 바뀌어서 처음에 카드 3장을 받는 것으로 변경된다면 card1, card2를 받게 되면 Hand 뿐만 아니라 User, Player까지 변경의 여파가 전이될 수 있는데, cards: List를 받게 된다면 Hand 내의 유효성 검증 로직만 수정하면 되기 때문에 좀 더 유지보수성이 좋다고 생각했습니다!  

cards: List를 file change가 1이 될 확률이 높으니 다른 사람이 코드리뷰하기도 좋을 것 같아요 🙂